### DISCUSSION TITLE
How can I stop a file upload by a name ?

### DISCUSSION THREAD

**Discussion by pullmann4rent** [remix-run/remix#5526] [Q&A]
Hello,

I can upload mutiple images. But I cant delete it when using <input type="file" multiple />.

So I created an array where I pass the deleted image names in there.

So in action I want to check the name in the upload handler but I dont have access to the names. How can I check by name ?

```
  const rq = request.clone();
  const formDatas = await rq.formData();

   const uploadHandler = unstable_composeUploadHandlers(
    async ({ name, data }) => {
        if (name !== "files") {
            return undefined
        }

      // CHECK BY NAME AND STOP

        console.log(data);

  
        const uploadedImage = await uploadImage(data);
       
        const imgUrl = uploadedImage.secure_url;
        return uploadedImage.secure_url;
    },
    unstable_createMemoryUploadHandler()
    );

    const formData = await unstable_parseMultipartFormData(
      request,
      uploadHandler
    );

    console.log(formData.get('name')); 
    
  return null;
```

**Comment by kiliman** [comment]
I'm not quite sure what you're trying to do here. By the time you get to your upload handler, the browser is already sending all the files. Where does the delete option come in? Couldn't you simply remove those items from the client before submitting them?

**Reply by shop-fluencer** [reply]
> I'm not quite sure what you're trying to do here. By the time you get to your upload handler, the browser is already sending all the files. Where does the delete option come in? Couldn't you simply remove those items from the client before submitting them?

Yes but how I remove it clientside?
